packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -77,6 +78,7 @@ export const ChatList = ({
         const client = new StatefulGraphChatListClient(props.chatThreadsPerPage);
         setChatListClient(client);
         setChatListState(client.getState());
+        setIsSignedIn(true);
```

Comment:
Do we need to handle the situation where the provider's state changed from "SignedIn" to something else? For example if it switches to "SignedOut", we may want to call `setIsSignedIn(false)`.

Reply:
Do we have a case for when the State is changed to SignedOut thou after a SignedIn? If a user explicitly signs out, I think it causes a redirect out to the Entra page.

Reply:
I don't have a specific use case for this. I just wasn't sure if there would ever be a case where the provider's state would change and the component's state be out of sync. I can't really think of a way to test this, so I will close out this comment.